libcaf_test/caf/test/runner.cpp

```diff
@@ -53,6 +53,24 @@ std::optional<unsigned> parse_log_level(std::string_view x) {
   return {};
 }
 
+std::optional<std::regex> to_regex(std::string_view regex_string) {
+#ifdef CAF_ENABLE_EXCEPTIONS
+  try {
+    return std::regex{regex_string.begin(), regex_string.end()};
+  } catch (std::regex_error& err) {
+    using detail::format_to;
+    auto errStream = std::ostream_iterator<char>{std::cerr};
+    format_to(errStream, "error while parsing argument '{}': {}\n",
```

Comment:
Aside from using the lambda here, please note that `errStream` is not according to our coding conventions (please use `snake_case`).

```suggestion
    format_to(err_out(), "error while parsing argument '{}': {}\n",
```